services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/DefaultS3CrtAsyncClient.java

```diff
@@ -160,10 +161,11 @@ public void afterMarshalling(Context.AfterMarshalling context,
                 SdkHttpExecutionAttributes.builder()
                                           .put(OPERATION_NAME,
                                                executionAttributes.getAttribute(SdkExecutionAttribute.OPERATION_NAME))
+                                          .put(CHECKSUM_SPECS,
+                                               executionAttributes.getAttribute(SdkExecutionAttribute.RESOLVED_CHECKSUM_SPECS))
```

Comment:
I see we do a 
```
getAttribute(SdkExecutionAttribute.RESOLVED_CHECKSUM_SPECS)
```
where is SdkExecutionAttribute.RESOLVED_CHECKSUM_SPECS  updated with resolved in case of code flow for transfer manager execution ?